cfme/tests/configure/test_access_control.py

```diff
@@ -836,22 +838,24 @@ def test_superadmin_tenant_project_crud(request):
         * Delete project
         * Delete tenant
     """
-    tenant = Tenant(
+    tc = TenantCollection(appliance=appliance)
+    pc = ProjectCollection(appliance=appliance)
```

Comment:
Same here, this seems to be going away fro ma nice tree structure.

Reply:
Changed